py3status/modules/xrandr_rotate.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+"""
+Switch between horizontal and vertical screen rotation on a single click.
+
+Configuration parameters:
+    - cache_timeout: how often to refresh this module.
+        default is 10
+    - format: a string that formats the output, can include placeholders.
+        default is '{icon}'
+    - horizontal_icon: a character to represent horizontal rotation.
+        default is 'H'
+    - horizontal_rotation: a horizontal rotation for xrandr to use.
+        available options: 'normal' or 'inverted'.
+        default is 'normal'
+    - vertical_icon: a character to represent vertical rotation.
+        default is 'V'
+    - vertical_rotation: a vertical rotation for xrandr to use.
+        available options: 'left' or 'right'.
+        default is 'left'
+
+Available placeholders for formatting the output:
+    - {icon} a rotation icon, specified by `horizontal_icon` or `vertical_icon`.
```

Comment:
Instead of the `{icon}` I'd be inclined to just have a `format_horizontal` and `format_vertical`  or maybe `format_h` and `format_v`.

Although it might be nice to spin the display all the way normal > 90 left > 180 > 90 right is there a reason for limiting this?